test/unittest/tests/wrappers/mock_connection.cpp

```diff
@@ -75,9 +75,8 @@ mock_connection::setup_block_manager(WT_SESSION_IMPL *session)
     WT_RET(__wt_spin_init(session, &_connection_impl->fh_lock, "file list"));
     WT_RET(__wt_spin_init(session, &_connection_impl->block_lock, "block manager"));
 
-    // Initialize an in-memory file system layer used for testing purposes.
-    F_SET(_connection_impl, WT_CONN_IN_MEMORY);
+    // Initialize a posix file system layer used for testing purposes.
```

Comment:
Unit tests run on windows too this will break. You'll need to do ifdef WIN else etc.